### ISSUE TITLE
read arguments from file

### ISSUE THREAD

**Issue by Kloenk** [rust-lang/rust-bindgen#1898]
rustc supports reading arguments from a file, if you give them via `@<filename>`.

I would love to see this feature in bindgen

**Comment by emilio** [comment]
Yes, this would be useful. We have an ad-hoc thing in Firefox for this: https://searchfox.org/mozilla-central/rev/d25eb00ab4e90cc0130cd18f303a04cc2a2f8409/servo/components/style/build_gecko.rs#27-40
